### ISSUE TITLE
Градиентный бустинг learning_rate

### ISSUE THREAD

**Issue by dmylnikov** [vkantor/MIPT_Data_Mining_In_Action_2016#21]
На каком шаге в градиентном бустинге добавляется коэффициент learning_rate: до последнего шага, в котором оптимизируется коэффициент перед добавляемым новым деревом или после этого шага?

**Comment by persiyanov** [comment]
Learning rate это и есть коэффициент перед добавляемым новым деревом.

**Comment by dmylnikov** [comment]
На последнем шаге бустинга мы побираем коэффициент a_i перед деревом b_i(x) (в обозначениях дз). Это не learning rate, learning rate - это константа, которая должна помогать от переобучения.

**Comment by senya-ashukha** [comment]
Его действительно можно подбирать, но это ведь не сильно важно для понимания. Используйте просто маленькую константу.

**Comment by senya-ashukha** [comment]
Конечно перед новым деревом.
